python/paddle/fluid/tests/unittests/dygraph_to_static/test_dict.py

```diff
@@ -139,5 +140,67 @@ def test_ast_to_func(self):
         self.assertTrue((self._run_dygraph() == self._run_static()).all())
 
 
+# Tests for dict pop
+@paddle.jit.to_static
+def test_dic_pop(x):
+    x = paddle.to_tensor(x)
+    dict_a = {"red": 0, "green": 1, "blue": 2}
+
+    m = dict_a.pop("red")
+    n = dict_a.pop("black", 3)
+
+    out = x + m + n
+    return out
+
+
+@paddle.jit.to_static
+def test_dic_pop_2(x):
+    x = paddle.to_tensor(x)
+    dict_a = {"red": x, "green": x + 1, "blue": x + 3}
+
+    m = dict_a.pop("red")
+    n = dict_a.pop("black", 3)
+
+    out = x + m + n
+    return out
+
+
+class TestDicPop(unittest.TestCase):
+    def setUp(self):
+        self.input = np.random.random((3)).astype('int32')
+        self.place = fluid.CUDAPlace(0) if fluid.is_compiled_with_cuda(
```

Comment:
Should we write test here using Paddle 2.0 API?

Reply:
Yes, thanks, done